### ISSUE TITLE
GUI Editor

### ISSUE THREAD

**Issue by thomthom** [thomthom/SKUI#27]
After #26 is complete it would be nice to have a Window editor. A visual way of creating controls in the window and then generate the Ruby code for it.

One could have one window acting as the host window where the controls are added, removed etc. And a toolkit window with a toolbar palette of controls to add and properties for the selected controls.
